database/src/main/resources/db/migration/V1.103.0__RHCLOUD-34487_add_endpoint_event_type_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE endpoint_event_type (
+         event_type_id UUID NOT NULL,
+         endpoint_id UUID NOT NULL,
+         CONSTRAINT pk_endpoint_event_type PRIMARY KEY (event_type_id, endpoint_id),
+         CONSTRAINT fk_endpoint_event_type_event_type_id FOREIGN KEY (event_type_id) REFERENCES event_type (id) ON DELETE CASCADE,
+         CONSTRAINT fk_endpoint_event_type_endpoint_id FOREIGN KEY (endpoint_id) REFERENCES endpoints (id) ON DELETE CASCADE
+) WITH (OIDS=FALSE);
```

Comment:
Since [PostgreSQL 8.1](https://www.postgresql.org/docs/8.4/runtime-config-compatible.html#GUC-DEFAULT-WITH-OIDS), the default value for ` default_with_oids` is `off`, so I don't think to explicitly state it.

Reply:
Ok great